fvm/state/accounts.go

```diff
@@ -282,11 +274,110 @@ func (a *Accounts) SetCode(address flow.Address, code []byte) error {
 		return nil
 	}
 
-	a.ledger.Set(string(address.Bytes()), string(address.Bytes()), keyCode, code)
+	return a.setValue(address, true, keyCode, code)
+}
+
+// GetStorageUsed returns the amount of storage used in bytes by this account
+func (a *Accounts) GetStorageUsed(address flow.Address) (uint64, error) {
+	storageUsedRegister, err := a.getValue(address, false, keyStorageUsed)
+	if err != nil {
+		return 0, err
+	}
+
+	if len(storageUsedRegister) != uint64StorageSize {
+		return 0, fmt.Errorf("account %s storage used is not initialized or not initialized correctly", address.Hex())
+	}
+
+	storageUsed := binary.LittleEndian.Uint64(storageUsedRegister)
+	return storageUsed, nil
+}
+
+func (a *Accounts) setStorageUsed(address flow.Address, used uint64) error {
+	buffer := make([]byte, uint64StorageSize)
+	binary.LittleEndian.PutUint64(buffer, used)
```

Comment:
Same as above, should maybe be big-endian, and just use an existing utility function